src/Components/Artwork/GridItem.tsx

```diff
@@ -37,7 +37,7 @@ export class ArtworkGridItem extends React.Component<Props, null> {
             style={{ position: "absolute", right: "10px", bottom: "10px" }}
           />
         </Placeholder>
-        <ArtworkMetadata artwork={artwork} />
+        <ArtworkMetadata extended artwork={artwork} />
```

Comment:
So, the `GridItem` is used in the `ArtworkGrid`, and thus is used in the artwork filters. I don't know why we wouldn't want to display the sale messaging, so this passes in true. This should fix the gene page.

Reply:
Do we even really need the `extended` option? If not, then I’d say let’s get rid of it so nobody else uses the component not realising they might need to specify this.

Reply:
Yea, we do need it- basically for the loyalty artworks we wanted the ability to use these components but hide the sale lines (as well as some other tweaks). So `extended` was used to signify that.

I think, if anything, it probably makes more sense for `extended=true` to be the default however, and override it in the loyalty artwork component.

Reply:
Agreed 👍